app/assets/stylesheets/home/components/_variables.scss

```diff
@@ -0,0 +1,11 @@
+$text-color: #0077B6;
+$breakpoint-desktop: 992px;
+
+@mixin desactivated_navbarMobile{
+  .navbar{
+    display: none;
+  }
+  .navbar-expand-lg .navbar-collapse{
+    display: none !important;
```

Comment:
Using `!important` is considered a bad practice because it makes difficult for other styles to apply correctly to the element. Almost anytime is used, the same result can be accomplished by paying attention to [selector specificity](https://developer.mozilla.org/es/docs/Web/CSS/Specificity). Always regard `!important` as a last resort.